packages/common/src/directives/ng_optimized_image/ng_optimized_image.ts

```diff
@@ -191,7 +203,13 @@ export const BUILT_IN_LOADERS = [imgixLoaderInfo, imageKitLoaderInfo, cloudinary
     '[style.position]': 'fill ? "absolute" : null',
     '[style.width]': 'fill ? "100%" : null',
     '[style.height]': 'fill ? "100%" : null',
-    '[style.inset]': 'fill ? "0px" : null'
+    '[style.inset]': 'fill ? "0px" : null',
```

Comment:
```suggestion
    '[style.inset]': 'fill ? "0" : null',
```
nit: units aren't necessary for zero

Reply:
Fixed